.circleci/config.yml

```diff
@@ -7,26 +7,31 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker
+      - run:
+          name: Install build packages
+          command: apk --no-cache add bash pcre-tools
       - run:
           name: Build docker images
           command: |
-            for APP in */; do
-              APP="${APP%/}"
-              cd $APP
-              docker build \
-                -t scalableminds/$APP:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM} \
-                -t scalableminds/$APP:${CIRCLE_BRANCH} \
-                .
-              cd ..
-            done
+            ./build.sh
       - run:
           name: Push docker images
           command: |
             docker login -u $DOCKER_USER -p $DOCKER_PASS
             for APP in */; do
               APP="${APP%/}"
-              docker push scalableminds/$APP:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM}
-              docker push scalableminds/$APP:${CIRCLE_BRANCH}
+              docker push scalableminds/$APP:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM} || {
+                docker tag \
+                  scalableminds/$APP:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM} \
+                  scalableminds/various:${APP}__${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM}
+                docker push scalableminds/various:${APP}__${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM}
+              }
```

Comment:
If the dockerhub repo does not exist, the image will be pushed to `various:APP__BRANCH[__BUILD]`. Is this desirable, @normanrz?